hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMMXBean.java

```diff
@@ -29,6 +29,8 @@ public interface OMMXBean extends ServiceRuntimeInfo {
 
   String getRpcPort();
 
+  String getOmRatisRoles();
```

Comment:
Nit: since `OMMXBean` is specific to OM, could be simplified to `getRatisRoles()` (which is consistent with `getRatisLogDirectory()` below).

Reply:
Changed, thanks for the suggestion !!